code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -0,0 +1,124 @@
+/obj/item/living_heart
+	name = "Living Heart"
+	desc = "Link to the worlds beyond."
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "living_heart"
+	w_class = WEIGHT_CLASS_SMALL
+	///Target
+	var/mob/living/carbon/human/target
+
+/obj/item/living_heart/attack_self(mob/user)
+	. = ..()
+	if(!IS_HERETIC(user))
+		return
+	if(!target)
+		to_chat(user,"<span class='warning'>No target could be found. Put the living heart on the rune and use the rune to recieve a target.</span>")
+		return
+	var/dist = get_dist(user.loc,target.loc)
+	var/dir = get_dir(user.loc,target.loc)
+
+	switch(dist)
+		if(0 to 15)
+			to_chat(user,"<span class='warning'>[target.real_name] is near you. They are to the [dir2text(dir)] of you!</span>")
+		if(16 to 31)
+			to_chat(user,"<span class='warning'>[target.real_name] is somewhere in your vicinty. They are to the [dir2text(dir)] of you!</span>")
+		if(32 to 127)
+			to_chat(user,"<span class='warning'>[target.real_name] is far away from you. They are to the [dir2text(dir)] of you!</span>")
+		else
+			to_chat(user,"<span class='warning'>[target.real_name] is beyond our reach.</span>")
+
+	if(target.stat == DEAD)
+		to_chat(user,"<span class='warning'>[target.real_name] is dead. Bring them onto a transmutation rune!</span>")
+
+/obj/item/melee/sickly_blade
+	name = "Sickly blade"
+	desc = "Twisted sickle with an ornamental eye. The eyes looks at you."
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "eldritch_blade"
+	inhand_icon_state = "eldritch_blade"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	flags_1 = CONDUCT_1
+	sharpness = IS_SHARP
+	w_class = WEIGHT_CLASS_NORMAL
+	force = 17
+	throwforce = 10
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "rended")
+
+/obj/item/melee/sickly_blade/attack(mob/living/M, mob/living/user)
+	if(!IS_HERETIC(user))
+		to_chat(user,"<span class='danger'>Echoes of the breat beyond break your mind!</span>")
+		var/mob/living/carbon/human/human_user = user
```

Comment:
what if a monkey or a dextrous stand is using this?

Reply:
Also, don't you mean beast or great rather than breat? If so it can go either way, from "Echoes of the great beyond breaks your mind!" or "Echoes of the great beast breaks your mind!"